rc-events-service/src/test/scala/com/epam/random_coffee/events/services/EventServiceImplSpec.scala

```diff
@@ -38,19 +45,19 @@ class EventServiceImplSpec extends AsyncWordSpec with AsyncMockFactory with OneI
 
     "update an event" when {
       "event exists" in {
```

Comment:
you should add more tests that will show what happens when you want don't provide all of the fields (e.g. `eventDate`)

Reply:
Added 4 tests for update with 1 parameter and also without it at all